### ISSUE TITLE
Force the user to enter a real salary when contributing

### ISSUE THREAD

**Issue by Zaker237** [osscameroon/jobsika#338]
As we can see on the App we have many unreal salaies such as 1 FCFA or 0 FCFA or even 190 FCFA
i suggest that we force the user to enter a salarie at least begger than 1000 FCFA, otherwise the form will not be validated.

**Comment by DipandaAser** [comment]
People can do entership and not being paid, those people would also like to rate the company

**Comment by Zaker237** [comment]
sure your are right
so what about to set it to either 0 FCFA or at least 1000 FCFA? because i believe the salaries like 190 or 350 FCFA that we habe online are just mistake. they likely wanted to type 190000 FCFA or 350 000 FCFA
@DipandaAser

**Comment by DipandaAser** [comment]
yeah that's cool @Zaker237 . don't forget to specify that to the user when he's filling the form.

**Comment by Zaker237** [comment]
The main problem hier is that these mistakes make the average salary going down.
Another solution will be to update the backend such that it multiplys the incoming salary by 1000 when the salary is smaller than 1000. 
with this when the user will enter 0 FCFA it will stay 0 and it will automatically handles the mistake of the users...

what do you thing @DipandaAser ??

**Comment by DipandaAser** [comment]
The first solution is better.
Just add it to the form validation
@Zaker237

**Comment by Zaker237** [comment]
oki nice @Asam237  you can go on

**Comment by Theryx** [comment]
What if we instead ask the user if he is sure about that input before validating. Kind of like a warning

**Comment by GenjiruSUchiwa** [comment]
> The main problem hier is that these mistakes make the average salary going down. Another solution will be to update the backend such that it multiplys the incoming salary by 1000 when the salary is smaller than 1000. with this when the user will enter 0 FCFA it will stay 0 and it will automatically handles the mistake of the users...
> 
> what do you thing @DipandaAser ??

Personally, I don't think it's a good idea to multiply salaries by 1,000 because you risk having amounts that are much too high and will distort the statistics;
I suggest to integrate a validation.
If the salary is between 0 and 50k we consider that it is an internship and we do not display the salary to the user and of course before validating the form we put a warning as @Theryx suggested

**Comment by pythonbrad** [comment]
> As we can see on the App we have many unreal salaies such as 1 FCFA or 0 FCFA or even 190 FCFA

Why not add a verification system ? Something like, the user should present a receipt.

**Comment by Zaker237** [comment]
Hallo,
@pythonbrad i get the point but it's a bit complicated due to the following points:

* First all we advocate and promise anonymity and therefore give us a copy of his pay slip to break this promisewe advocate and promise anonymity and therefore give us a copy of his pay slip to break this promise
* extracting data from a pay sheet for comparison with other users is a very complex task requiring resources that we do not have yet
* we also need to make sure that contributing is as easy as possible for users

**Comment by Theryx** [comment]
Hi, @pythonbrad thank you for contributing to this issue. I appreciate you taking the time to help us brainstorm on this issue.
I get the point you are trying to bring here and it could be interesting if we had the resources to build that.

I thank the solution you trying to bring is related to salary verification. That is you are asking the contributor a prove of his salary. On the other hand, what we are trying to solve here is wrong salary input(usually happens when a contributor validates a salary without verifying the correctness of the figures entered in the field)

For the above reason, your solution might not be the appropriate solution for this context
